mautic/mautic
# [UI] Replace most of FA icons (2)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🟢
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR replaces more icons recursively.

Next steps/PRs:
- Replace all of FA icons (3)
- Remove all remaining FA utility classes
- Remove the FA files from CoreBundle

---

(ignore this)
Regular expression for search
`^(?!.*(spinner|ri-)).*fa-.*$`






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the interface and check if everything is normal/all icons are there
